### PR TITLE
fix #1302: add 'stay connected' checkbox

### LINKED ISSUE #1302: [v2] "stay connected" checkbox

### PR DESCRIPTION
**Pull request by nicosomb**

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/views/themes/material/Security/login.html.twig [hunk 1]
```diff
@@ -33,6 +33,11 @@
                             <input type="password" id="password" name="_password" />
                         </div>
 
+                        <div class="input-field s12">
+                            <input type="checkbox" id="remember_me" name="_remember_me" checked />
+<label for="remember_me">Keep me logged in</label>
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/Security/login.html.twig]: Indentation fail :)

**Comment by nicosomb** [comment]
fixed.

**Comment by j0k3r** [comment]
What about translation ?

**Comment by nicosomb** [comment]
grrr, forgot it. will fix.

**Comment by nicosomb** [comment]
Fixed.

**Pull request merged**